kava-stats.js

```diff
@@ -530,15 +530,6 @@ const mapCoinGeckoApiData = async (coinGeckoApiJson) => {
   return coinGeckoApiJson.market_data.price_change_percentage_24h;
 }
 
-const setSwpSupplyAmount = async (supplyTotalJson) => {
-  const swpSupplyAmount = Number(supplyTotalJson.result.find(coin => coin.denom === 'swp').amount);
-
-  return {
-    denom: 'swp',
-    amount: swpSupplyAmount
-  }
-};
```

Comment:
Redundant since `supplyTotalJson` pulls from [api.kava.io/supply/total](api.kava.io/supply/total) which includes SWP when passed to `mapSuppliedAmounts` on line 915.